sdk/digitaltwins/digitaltwins/review/digitaltwins.api.md

```diff
@@ -0,0 +1,372 @@
+## API Report File for "@azure/digitaltwins"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureServiceClientOptions } from '@azure/core-arm';
+import * as coreHttp from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface AzureDigitalTwinsAPIOptions extends AzureServiceClientOptions {
+    // (undocumented)
+    baseUri?: string;
+}
+
+// @public
+export interface DigitalTwinModelsAddOptionalParams extends coreHttp.RequestOptionsBase {
+    models?: any[];
+}
+
+// @public
+export type DigitalTwinModelsAddResponse = Array<ModelData> & {
+    _response: coreHttp.HttpResponse & {
```

Comment:
do you want to expose `_response`? We tend to not expose the raw response for most data plane operations

Reply:
Isn't it a requirement in Track 2 to always expose the full HTTP response for debug purposes?

---
In reply to: [447255889](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447255889) [](ancestors = 447255889)

Reply:
`_response` is good.